packs/trello/README.md

```diff
@@ -47,6 +47,39 @@ To obtain an oAuth token, refer to the documentation at https://trello.com/docs/
 +---------------------------+--------+--------------------+------------------------------------------------------+
 ```
 
+## Sensors
+### TrelloListSensor
+Listens for the new Actions (changes) in Trello List(s) and dispatches trigger for each new event occurred.
+
+#### Config
+Parameters in `list_actions_sensor` for every trello list:
+* `list_id` - Trello List ID we monitor for new actions (required)
+* `board_id` - Board ID where Trello List is located (required)
+* `api_key` - Trello API key (optional)
+* `token` - Trello API token (optional)
+* `filter` - Filter actions by type(s) (eg. createCard, deleteCard) (optional)
+
+For list of available filters see [Trello API docs](https://trello.com/docs/api/list/index.html#get-1-lists-idlist-actions).
+
+> API credentials work at any level with lower priority for top-level config credentials:
+`list config` > `lists config` > `global config`.
+It means that every Trello list can have its own unique credentials to work with different accounts.
+
+See [config](config.yaml) for more examples.
+Keep in mind, that Sensor with invalid config (bad credentials) will die after 3 connection retries.
+
+#### Output
+When `TrelloListSensor` receives new data, it emits:
+* trigger: `trello.new_action`
+* and returns data:
+  * `trigger.id` - Action ID (string)
```

Comment:
Is the list id also included in the payload?

Since we support monitoring multiple lists I think it should be, but that's not visible in the code because we directly dispatch the payload as returned by the API.